htdocs/cron/admin/cron.php

```diff
@@ -44,7 +44,7 @@
 
 	$db->begin();
 
-	$i += dolibarr_set_const($db, 'CRON_KEY', GETPOST("CRON_KEY"), 'chaine', 0, '', 0);
+	$i += dolibarr_set_const($db, 'CRON_KEY', GETPOST("CRON_KEY"), 'chaine', 0, '', $conf->entity);
```

Comment:
Don't understand how this can work because into the linux crontab, we have only one entry to execute the cron taks. Then the CLI script process each entity separately but the triggering is common to the dolibarr installation.